tests/TASVideos.MovieParsers.Tests/TASVideos.MovieParsers.Tests.csproj

```diff
@@ -28,6 +28,11 @@
 		<EmbeddedResource Include="WtfSampleFiles\*.wtf" />
 	</ItemGroup>
 
+	<ItemGroup>
+	  <None Remove="LtmSampleFiles\ruffle-windows-override.ltm" />
+	  <None Remove="LtmSampleFiles\ruffle.ltm" />
+	</ItemGroup>
+
```

Comment:
Why are these lines needed when they aren't needed for all other files?

Reply:
IDE did that, I don't know

Reply:
`<None/>` items are essentially hints to the IDE. The SDK automatically includes anything that's not a source file (`.cs`, `.resx`, etc.) in `@(None)` so they'll show up in the IDE's project tree view. Removing files like this hides them from the tree, but makes no difference to the compilation.

Reply:
just remove the lines, visual studio adds them but we don't want them

Reply:
Removed